rest/changes_test.go

```diff
@@ -273,3 +274,50 @@ func TestWebhookWinningRevChangedEvent(t *testing.T) {
 	assert.Equal(t, 5, int(atomic.LoadUint32(&WinningRevChangedCount)))
 	assert.Equal(t, 6, int(atomic.LoadUint32(&DocumentChangedCount)))
 }
+
+// TestCurrentVersionPopulationOnChannelCache:
```

Comment:
This test should be moved to the db package - I don't see anything here that merits making the channelCache accessible from outside the db package.  In general that kind of requirement makes me think I've got the test in the wrong location, unless there's something in the rest package that also needs to be exercised.  That doesn't seem like the case here.
Have a look at the tests in db/changes_test.go (like TestFilterToAvailableChannels) for an example of a test that's just trying to write a doc and check it arriving over DCP.